kserve-modelmesh.yaml

```diff
@@ -30,23 +30,28 @@ pipeline:
       export JAVA_HOME=/usr/lib/jvm/java-17-openjdk
       mvn -B package -Dfile.encoding=UTF8 -DskipTests=true --file pom.xml
       mkdir -p ${{targets.destdir}}/opt/kserve/mmesh
-      mv /home/build/target/dockerhome ${{targets.destdir}}/opt/kserve/mmesh/
-      chmod -R 775 ${{targets.destdir}}/opt/kserve/mmesh/dockerhome
+      mv /home/build/target/dockerhome/* ${{targets.destdir}}/opt/kserve/mmesh/
+
+      echo "$(date +%Y%m%d)-$(git rev-parse --short HEAD)" > ${{targets.destdir}}/opt/kserve/mmesh/build-version
+      mkdir -p ${{targets.destdir}}/etc
+      mkdir -p ${{targets.destdir}}/opt/kserve/mmesh/log
+      chmod -R 775 ${{targets.destdir}}/opt/kserve/mmesh/
+      touch ${{targets.destdir}}/etc/environment
```

Comment:
can we pull in `linux-pam` as runtime deps instead of creating this file. 

$ apk info -W /etc/environment
/etc/environment is owned by linux-pam-1.6.1-r1